adm/src/blockstore.rs

```diff
@@ -0,0 +1,122 @@
+use protobuf;
```

Comment:
Need copyright statement at beginning of all files.

Reply:
woops